server/app/views/applicant/ApplicantUpsellTemplate.html

```diff
@@ -124,6 +121,10 @@ <h2 th:text="#{content.otherProgramsToApplyFor}"></h2>
           ></div>
         </div>
       </section>
+
+      <div th:unless="${showProgramsCardsSection}" class="page-spacing">
+        <!-- Add spacing only when the 'Program Cards' section is hidden, since that section adds its own bottom margins. !-->
```

Comment:
(blocking) Please swap over to use [Thymeleaf parser-level comment blocks](https://www.thymeleaf.org/doc/tutorials/3.1/usingthymeleaf.html#thymeleaf-parser-level-comment-blocks). It prevents them from being rendered to the client.

Reply:
Done. Is there a max char per line? I only found bin/fmt for formatting, but that doesn't include HTML

Reply:
Unless the formatter makes changes, do what feels best.